edx/xblock-lti-consumer
# [BD-2][BB-2365] LTI 1.3 Gating and documentation

This PR gates the LTI 1.3 features, fixes a few minor bugs/rebase issues and adds documentation related to the LTI 1.3 integration.

**Testing instructions:**
1. Checkout this branch and install it on your master devstack.
2. Follow the updated README guide to set up an LTI 1.3 test tool.
3. Check that you can get a successful LTI 1.3 Launch.
4. Modify the Client ID in the LTI tool and check that an error page is displayed.
5. Check that the added documentation is ok.

**Reviewers:**
- [x] @__xitij2000__
